packages/website/ts/pages/governance/governance.tsx

```diff
@@ -0,0 +1,299 @@
+import { Heading, Paragraph } from 'ts/components/text';
+import { Countdown } from 'ts/pages/governance/countdown';
+import { ModalVote } from 'ts/pages/governance/modal_vote';
+import { RatingBar } from 'ts/pages/governance/rating_bar';
+import { VoteInfo, VoteValue } from 'ts/pages/governance/vote_form';
+import { VoteStats } from 'ts/pages/governance/vote_stats';
+import { colors } from 'ts/style/colors';
+import { configs } from 'ts/utils/configs';
+import { utils } from 'ts/utils/utils';
+
+interface LabelInterface {
+    [key: number]: string;
+}
+
+export interface TallyInterface {
+    zeip?: string;
+    yes?: BigNumber;
+    no?: BigNumber;
+    blockNumber?: string;
+    totalVotes?: string;
+    totalBalance?: BigNumber;
+}
+
+interface State {
+    isContactModalOpen: boolean;
+    isVoteModalOpen: boolean;
+    isWalletConnected: boolean;
+    isVoteReceived: boolean;
+    providerName?: string;
+    tally?: TallyInterface;
+}
+
+const benefitLabels: LabelInterface = {
+    1: 'Little Benefit',
+    2: 'Medium Benefit',
+    3: 'High Benefit',
+};
+
+const riskLabels: LabelInterface = {
+    1: 'Low Risk',
+    2: 'Medium Risk',
+    3: 'High Risk',
+};
+
+const proposalData = {
+    zeipId: 23,
+    title: 'ZEIP23: Trading Bundles of Assets',
+    summary: `This ZEIP introduces the MultiAssetProxy which brings support for trading arbitrary bundles of assets in 0x protocol. Historically only a single asset could be traded per each side of a trade. With the introduction of the MultiAssetProxy, users will be able to trade multiple ERC721 assets and even mix ERC721 and ERC20 to trade in a single order.`,
+    url: 'https://blog.0xproject.com/zeip-23-trade-bundles-of-assets-fe69eb3ed960',
+    votingDeadline: 1551584000,
+    benefit: {
+        title: 'Benefit',
+        summary: `Supporting bundled trades is one of the most commonly requested features since the launch of 0x v2. Demand originated from our discussions with gaming and NFT related projects, but this upgrade provides utility to Prediction Markets as well as any relayer that wants to offer baskets of tokens. The MultiAssetProxy will enable new possibilities of trading.`,
+        rating: 3,
+        links: [
+            {
+                text: 'Learn More',
+                url: 'https://0xproject.quip.com/k1ERAXnUj2ay/ZEIP23-Support-for-MultiAssetProxy',
+            },
+            {
+                text: 'Technical detail',
+                url: 'https://github.com/0xProject/ZEIPs/issues/23',
+            },
+        ],
+    },
+    risks: {
+        title: 'Risk',
+        summary: `Deploying the MultiAssetProxy is a hot upgrade to 0x protocol where the state of active contracts is modified. The contracts being modified contain allowances to users tokens. As such the MultiAssetProxy has successfully undergone a third party audit. We encourage the community to verify the MultiAssetProxy code along with the state changes.`,
```

Comment:
`Deploying the MultiAssetProxy is a hot upgrade that requires modifying the state of existing contracts within 0x protocol. The contracts being modified have already been approved to transfer tokens on behalf of many users. Therefore, it is possible for bugs within the MultiAssetProxy to result in the loss of user funds. However, the code of the MultiAssetProxy is relatively straightforward and has successfully undergone a full third party audit. We encourage the community to verify the code as well as the state changes.`

Reply:
I found this line a little awkward:
> The contracts being modified have already been approved to transfer tokens on behalf of many users.
I prefer the original:
> The contracts being modified contain allowances to [many] users tokens.